packages/css-library/src/utilities/text-decoration.scss

```diff
@@ -0,0 +1,14 @@
+@use 'uswds-core/src/styles/functions' as *;
+@use '../tokens/text-decoration' as *;
+@use './settings' as *;
+
+$u-text-decoration: (
+  text-decoration: (
+    base: 'vads-text',
```

Comment:
And then:

- `vads-text-none`
- `vads-text-underline`
- `vads-text-strikethrough`